### PR TITLE
[T4A7][F10-B2]Joel Foo

### PR DESCRIPTION
**Pull request by jaeoheeail**
Ready for review.

### CODE REVIEW

File: src/seedu/addressbook/data/person/Name.java [hunk 1]
```diff
@@ -11,54 +11,78 @@
  */
 public class Name {
 
-    public static final String EXAMPLE = "John Doe";
-    public static final String MESSAGE_NAME_CONSTRAINTS = "Person names should be spaces or alphabetic characters";
-    public static final String NAME_VALIDATION_REGEX = "[\\p{Alpha} ]+";
+	public static final String EXAMPLE = "John Doe";
+	public static final String MESSAGE_NAME_CONSTRAINTS = "Person names should be spaces or alphabetic characters";
+	public static final String NAME_VALIDATION_REGEX = "[\\p{Alpha} ]+";
```
Comment (KulaniT) [inline src/seedu/addressbook/data/person/Name.java:17]: why public?

**Comment by Candiie** [comment]
Hi Joel ~ As mentioned in the tutorial, those are some awesome commits you have, but **do note that git only show the first sentence as the main commit message** ~ so do summaries your implementation following by adding details like what you've did after the main messages **to make your commit messages more meaningful**. :)

**Comment by jaeoheeail** [comment]
@Candiie Noted! Will improve on my commit messages

File: src/seedu/addressbook/data/person/NameTest.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
+package seedu.addressbook.data.person;
+
+import static org.junit.Assert.*;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
```
Comment (KulaniT) [inline src/seedu/addressbook/data/person/NameTest.java:10]: Header comments missing

File: src/seedu/addressbook/data/person/NameTest.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
+package seedu.addressbook.data.person;
+
+import static org.junit.Assert.*;
```
Comment (KulaniT) [inline src/seedu/addressbook/data/person/NameTest.java]: Please avoid importing all classes. Instead import only the required classes.

**Comment by KulaniT** [comment]
@jaeoheeail  : Some comments added. Please close the PR after reading comments

**Pull request closed without merging**